modules/storage-expanding-csi-volumes.adoc

```diff
@@ -20,7 +22,7 @@ CSI volume expansion does not support the following:
 
 * Dynamic provisioning is used.
 
-* The controlling `StorageClass` object has `allowVolumeExpansion` set to `true` (see xref:../storage/expanding-persistent-volumes.adoc#add-volume-expansion_expanding-persistent-volumes[Enabling volume expansion support]).
+* The controlling `StorageClass` object has `allowVolumeExpansion` set to `true`. For more information, see Enabling volume expansion support. 
```

Comment:
I'd recommend putting it in italics or quotes (don't know if we have a ruling on which).